idaes/models/properties/modular_properties/state_definitions/FTPx.py

```diff
@@ -381,6 +381,12 @@ def state_initialization(b):
     henry_comps = []
     init_VLE = False
     for pp in _pe_pairs:
+        # Calculate states at subcritical pressure
+        if b.params.config.supercritical_extension == True:
```

Comment:
Does this need to be added to the other state definitions as well?

Reply:
I don't think so, the other state definitions do not have this calculation. I will test regardless, but hasn't didn't showed up as a problem so far.